src/js/components/action.jsx

```diff
@@ -17,6 +18,23 @@ export class ActionComponent extends Component {
         };
     }
 
+    onPostbackClick = () => {
+        this.setState({
+            state: 'processing'
+        });
+
+        postPostback(this.props._id).then(() => {
+            this.setState({
+                state: 'paid'
```

Comment:
Should change those to be more general e.g. `done` and `normal` instead of `paid` and `offered`